src/lib/viewers/doc/_docBase.scss

```diff
@@ -16,6 +20,10 @@ $thumbnail-sidebar-width: 226px;
         transform: translateX(-$thumbnail-sidebar-width);
         transition: transform 300ms cubic-bezier(.4, 0, .2, 1);
         width: $thumbnail-sidebar-width;
+
+        @include for-phone-only {
```

Comment:
Rather than overriding styles, could we wrap all the affected classes in a min-width media query? That way we don't need to maintain the base/overrides separately. Something like this:

```
@mixin breakpoint-medium {
   @media (min-width: 600px) { @content; }
}

.bp {
    @include breakpoint-medium {
        .bp-thumbnails-container { // etc }
    }
}
```